src/Controller/NphPDFController.php

```diff
@@ -26,4 +26,20 @@ public function render_pdf(
         $OrderPDF = $PDF->batchPDF($nphOrderService->getParticipantOrderSummaryByModuleAndVisit($participantId, $module, $visit)['order'], $nphParticipantSummaryService->getParticipantById($participantId), $module, $visit);
         return new Response($OrderPDF, Response::HTTP_OK, ['content-type' => 'application/pdf']);
     }
+
+    /**
+     * @Route("/nph/participant/{participantId}/render_pdf/module/{module}/visit/{visit}/{sampleGroup}", name="nph_render_pdf_sample_group")
+     */
+    public function renderPDFSampleGroup(
+        $participantId,
+        $module,
+        $visit,
+        $sampleGroup,
+        PDFService $PDF,
+        NphOrderService $nphOrderService,
+        NphParticipantSummaryService $nphParticipantSummaryService
+    ): Response {
+        $OrderPDF = $PDF->batchPDF($nphOrderService->getParticipantOrderSummaryByModuleVisitAndSampleGroup($participantId, $module, $visit, $sampleGroup)['order'], $nphParticipantSummaryService->getParticipantById($participantId), $module, $visit);
```

Comment:
It's hard to read the logic when the parameters span multiple lines.  Break this into separate lines to make it easier to read. 

```suggestion
        $OrderPDF = $PDF->batchPDF($nphOrderService->
            getParticipantOrderSummaryByModuleVisitAndSampleGroup(
                $participantId, 
                $module, 
                $visit,
                $sampleGroup)['order'],
                $nphParticipantSummaryService->getParticipantById($participantId), 
                $module, 
                $visit
        );
```